### PR TITLE
Fix Translation Warnings on Home #302

### PR DESCRIPTION
**Pull request by AndreaGon**
This should fix Issue #302.

### CODE REVIEW

File: js/homeview.js [hunk 1]
```diff
@@ -233,7 +233,7 @@ enyo.kind({
 
 	localize: function() {
 		if (this.otherview && this.otherview.localize) {
-			this.otherview.localize();
+			this.otherview.localize;
```
Comment (llaske) [inline js/homeview.js]: Very strange fix: removing parenthesis is equivalent to don't call the function at all.

File: js/homeview.js [hunk 2]
```diff
@@ -732,7 +732,7 @@ enyo.kind({
 
 	rendered: function() {
 		this.inherited(arguments);
-		this.localize();
+		this.localize;
```
Comment (llaske) [inline js/homeview.js]: Very strange fix: removing parenthesis is equivalent to don't call the function at all.

File: js/homeview.js [hunk 3]
```diff
@@ -743,7 +743,7 @@ enyo.kind({
 		this.$.neighborbutton.setNodeProperty("title", l10n.get("NeighborhoodView"));
 		this.$.helpbutton.setNodeProperty("title", l10n.get("Tutorial"));
 		if (app.localize) {
-			app.localize();
+			app.localize;
```
Comment (llaske) [inline js/homeview.js]: Very strange fix: removing parenthesis is equivalent to don't call the function at all.

**Comment by AndreaGon** [comment]
I restored the parenthesis in this latest commit. Let me know if there are still some issues.

File: js/homeview.js (hunk 3 above)
Comment (AndreaGon) [reply js/homeview.js]: Restored parenthesis

**Comment by AndreaGon** [comment]
Please check @llaske

**Comment by llaske** [comment]
That's better but tooltips on buttons (Help, Neighborhood, ...) is missing now.

**Comment by AndreaGon** [comment]
@llaske Are the tooltips only missing when the screen size is small? When I tried it on a normal window size, the tooltips for the buttons show themselves properly.

**Comment by llaske** [comment]
Hmmm, you're right, the tooltip disappear when the window is resized then do not appear again even if the window is enlarged.

**Comment by AndreaGon** [comment]
@Ilaske I guess it's normal for the title tooltip on buttons to disappear when the screen size is small as I do not think that mobile browsers support this, as I tried restoring the original code and the output is the same as the modified code. Also, the tooltip seemed to work properly if the window is enlarged again. What do you think?

**Comment by llaske** [comment]
Yes, you're right.
By the way I've found a side effect of your fix: tooltip and search label disappear when coming from the neighborhood view.
Steps to reproduce:

- Go to the neighborhood view and join an existing activity (created by another user)
- Stop the activity
- Go to the home view
- The tooltip and search label has disappeared 

If you test the same thing on dev.sugarizer.org, it works correctly.
Not proud of that but there is lot of imbrication on the localization functions. Thanks for your help on this.

**Comment by AndreaGon** [comment]
Hello!
Based on the PR on the webL10n library, loading and parsing the locale.ini file is happening after webL10n starts localizing. In my recent PR, I tried a workaround suggested in the webL10n PR, which worked for Sugarizer. I would appreciate your opinion.

**Comment by llaske** [comment]
Hmmm. Interesting but it will replace a (bunch of) warning by another:

> webL10n.js:124 [Deprecation] Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check https://xhr.spec.whatwg.org/.

Considering that translation warnings could be ignored because the UI is refreshed when the localization is loaded, I wonder if the good solution is not just to remove log by setting `gDEBUG` to 0 [here](https://github.com/llaske/sugarizer/blob/master/lib/webL10n.js#L57).

What do you think?

**Comment by AndreaGon** [comment]
Setting the variable gDEBUG to 0 does remove the translation warnings. However, it might be harder to see possible future localization bugs with gDEBUG turned off. In my opinion, the translation warnings on home should just be ignored, as it doesn't have any bad effects in Sugarizer. 

Thoughts? @llaske

**Comment by llaske** [comment]
Okay. Thanks for your contribution.

**Pull request closed without merging**